src/containers/SubjectPage/components/SubjectEditorChoices.tsx

```diff
@@ -50,17 +50,14 @@ const SubjectEditorChoices = ({
     .filter(x => x !== null)
     .filter(hasContentUri)
     .map((resource: GQLTaxonomyEntity) => ({
+      id: resource.meta ? resource.meta.id.toString() : '',
       title: resource.name,
-      image:
-        resource.meta && resource.meta.metaImage
-          ? resource.meta.metaImage.url
-          : '',
+      text: resource.meta?.metaDescription ?? '',
       type: getResourceTypeName(resource, t),
-      id: resource.meta ? resource.meta.id.toString() : '',
-      text: resource.meta ? resource.meta.metaDescription : '',
+      image: resource?.meta?.metaImage?.url,
```

Comment:
Defaulten var en tom streng, nå er det `undefined`, vet ikke åssen det kommer til å se ut/påvirke funksjonaliteten.

Reply:
Dersom den er undefined så brukes tom streng i SubjectCarousel. Denne komponenten vises ikkje i dag uansett, men bør ikkje slettes fordi vi kan redigere data for dette i ed.